package.json

```diff
@@ -1 +1,9 @@
-{}
\ No newline at end of file
+{
```

Comment:
This package.json is not in the correct location. You should update existing package.json in examples/bri-3/